doc/manual/parallel-computing.rst

```diff
@@ -451,36 +550,33 @@ preemptively. This means context switches only occur at well-defined
 points: in this case, when :func:`remotecall_fetch` is called.
 
 
-Channels
```

Comment:
It occurs to me that we may eventually want to have `AbstractTask` which `Task` and `RemoteTask` are concrete subtypes of.